client/v2/client.go

```diff
@@ -191,14 +196,14 @@ func NewUDPClient(conf UDPConfig) (Client, error) {
 		return nil, err
 	}
 
-	payloadSize := conf.PayloadSize
-	if payloadSize == 0 {
-		payloadSize = UDPPayloadSize
+	udpPayloadSize := conf.PayloadSize
```

Comment:
Renaming this local variable to have a `udp` prefix is unnecessary. It's already in a function called `NewUDPClient` and it used to set the `payloadSize` field in a `udpclient` struct. Revert this.